dashboard/src/components/injector/che-injector.service.js

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+
+/**
+ * This class is handling the service for the additional element injection.
```

Comment:
This class is handling the service for the injecting UI elements to other widgets from outside